index.bs

```diff
@@ -604,6 +613,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|.
                 :   {{AuthenticatorAttestationResponse/attestationObject}}
                 ::  |attestationObject|
+            :   {{ScopedCredential/clientExtensionResults}}
+            ::  A new {{AuthenticationExtensions}} object containing the [=extension identifier=] → [=client extension output=] entries
+                created by running each extension's [=client extension processing=] algorithm to create the [=client extension outputs=],
+                for each [=client extension=] in {{AuthenticatorResponse/clientDataJSON}}.clientExtensions.
```

Comment:
Ok, that's cool. It looks like some places in the spec also expect to return authenticator extension outputs in the client extension output, but because this step calls the "client extension processing" algorithm, which in all the authenticator extensions only specifies how to turn client inputs into authenticator inputs, it doesn't actually get the outputs transformed.

Looking at the text in [Client extension processing](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#sctn-client-extension-processing) that

> Extensions that require authenticator processing MUST define … the process by which the authenticator extension output can be used to determine the client extension output.

it seems like extensions were supposed to define what this step expects, but for example [txAuthSimple](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#sctn-simple-txauth-extension), specifies this transformation in the "Client extension output" section instead.

This might tie together better if this step called the extension's [=client extension output=] algorithm, which can use both the client input and authenticator output as inputs?